branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -0,0 +1,54 @@
+package com.virtuslab.branchlayout.api;
+
+import io.vavr.Tuple;
+import io.vavr.collection.List;
+import io.vavr.collection.Map;
+import io.vavr.control.Option;
+import lombok.Getter;
+import org.checkerframework.checker.interning.qual.UsesObjectEquals;
+
+@UsesObjectEquals
+public class BranchLayout implements IBranchLayout {
```

Comment:
Possibly `IBranchLayout` can be completely removed now that `BranchLayout` is in API... not sure yet